src/runtime/lpython_parser.py

```diff
@@ -35,6 +35,8 @@ def visit_Constant(self, node):
                         node.value.imag, node.kind)
         elif isinstance(node.value, dict):
             return python_ast.Dict(node.keys, node.values)
+        elif isinstance(node.value, tuple):
+            return python_ast.Tuple(node.elts)
```

Comment:
The `Tuple` node is defined as:
```
         | Tuple(expr* elts, expr_context ctx)
```
Isn't it missing an argument?

Reply:
Sorry, I missed it.  Thanks for pointing!